### DISCUSSION TITLE
是不是 管理被请去喝茶了都8天没有更新了 不是有很多管理员的吗？？？？？？？？？？？？？？？？？？？？？？？

### DISCUSSION THREAD

**Discussion by ewkyrtas** [WeNeedHome/SummaryOfLoanSuspension#1136] [General]
**是不是 管理被请去喝茶了都8天没有更新了 不是有很多管理员的吗？？？？？？？？？？？？？？？？？？？？？？？**

**Comment by ewkyrtas** [comment]
释放农民消费潜力、鼓励农民进城购房再次成为全国多地推动楼市发展的一大举措。

　　据澎湃新闻不完全统计，今年以来，至少包括吉林省延吉、吉林省吉林、江苏省泰州、湖北省黄冈、贵州省遵义、安徽省潜山、黑龙江省佳木斯、吉林省长春等城市以及四川省凉山彝族自治州宁南县、贵州省黔西南州晴隆县、江苏盐城阜宁县等地出台鼓励农民进城购房的相关政策。

　　同时，可以看到，在出台鼓励农民进城购房的城市中，地方政府多给予购房补贴、契税补贴等优惠政策。

![](https://pics2.baidu.com/feed/58ee3d6d55fbb2fb4bf35288503ff0ae4423dccb.jpeg?token=f73190595d346f45b50a80620792a8a7)

　　多地财政补贴，支持农民进城购房

　　以吉林省长春为例，今年8月11日，长春市住房保障和房屋管理局发布《关于进一步给予进城农民和来（留）长人才购房补贴政策支持的通知》。《通知》显示，对购买符合条件的新建商品住房给予200元/平方米购房补贴。

　　按照要求，申请人与房地产开发企业签订的正式商品房买卖合同网签备案时间在2022年8月1日至10月8日之间；所购房屋需坐落在长春市主城区及开发区国有土地上，不含双阳区、九台区及各县（市）；房屋建筑面积在90平方米以下；房屋性质仅限新建商品住房，不包括商业、办公等非住宅类房屋、别墅、保障性住房以及二手住房；所购房屋需为申请人个人首套唯一住房。

　　除了长春市，吉林省多个城市亦发布相关政策，包括延吉市、吉林市等。

　　其中，延吉市明确，对于农民购买家庭（家庭成员范围包括购房人、配偶以及未成年子女）唯一住房，政府参照购房农民所缴纳契税的额度全额给予补贴。

　　吉林市的政策显示，农民或大学生购买吉林市城市规划区内国有土地上90平方米及以下新建商品住房（不包括别墅类住房，下同），政府每套补贴0.6万元；购买90平方米以上新建商品住房，政府每套补贴1万元。

　　另据《佳木斯日报》报道，今年7月26日，佳木斯市人民政府出台十条优惠政策，其中提及农民进城购房补贴。在佳木斯市市区购买新建商品住房的农民购房人，每平方米补贴100元，每套补贴总额不超过2万元；购买新建商业用房的，每平方米补贴150元，每套补贴总额不超过3万元。

　　湖北黄冈出台的《关于进一步促进黄冈市区房地产市场平稳健康发展的补充措施》明确，实行农村户籍人口进城购房补贴，农民进城购房一次性补贴1万元（落户黄冈城区）。

　　诸葛找房数据研究中心高级分析师陈霄指出，近期以来，多城发布鼓励农民进城购房的政策，如遵义、盐城阜宁县以及长春等，出台的政策内容多为提供购房补贴和优惠等形式，从出台的城市来看，以一些库存压力大、去化困难的弱二线和三四线城市为主，出台这样的政策也是为了增加购房需求，缓解库存压力，提振市场信心。将政策聚焦于农民这样庞大的群体，有利于提升农民进城置业的积极性，一定程度上能带动一部分需求释放。

　　鼓励农民进城购房对楼市效果大吗？

　　可以看到的是，鼓励农民进城购房并非此次楼市调控的新鲜产物。

　　早在2015年底，中共中央政治局召开会议，会议提出，要化解房地产库存，通过加快农民工市民化，推进以满足新市民为出发点的住房制度改革，扩大有效需求，稳定房地产市场。要扩大有效供给，保持有效投资力度，着力补齐短板。

　　在彼时“去库存”的大背景下，加快农民工市民化，鼓励有条件的农民工买房成为重要措施。

　　按照新华网2016年2月发布的文章，当时山西、山东、甘肃等省明确出台政策鼓励引导农民进城购房。如山东将对农民进城购房给予契税补贴、规费减免、贷款贴息、物业费补贴等政策，以使其成为长期稳定的住房消费群体。

　　2016年2月19日，中国农业银行还曾推出“农民安家贷”产品，该产品是国内首个专门针对农民群体的个人住房贷款产品，致力于满足广大农民进城购买商品住房的需求。

　　从效果来看，根据新华网报道，截至2016年7月31日，安徽农行“农民安家贷”实现发放23329笔，发放金额69.91亿元，帮助农民进城购房。

　　安徽地区房地产“去库存”效果也得以凸显。以六安市为例，2015年市区待售商品房面积达663万余平方米，2016年一季度，六安市区销售商品房120多万平方米，同比翻了一番，其中农民购房占比超过50%，去化周期由28个月缩短到18个月。

　　在58安居客房产研究院分院院长张波看来，2016年大量三四线城市开始去库存，就已经消耗了大量的农民在县城的买房需求，同时前几年返乡置业更是吸引了一大批在三四线老家买房年轻人的购房需求。因此，本轮的鼓励农民进城买房能够有效吸引到的购买力和几年前相比已大大减少。

　　张波指出，前一轮去库存后，由于三四线城市也紧随出现房价上涨，最终导致市场的热度明显上升，并带动房企在中小城市拿地热情同步上扬，所导致的结果就是三四线城市的库存量又呈现上升态势，加之市场降温的大背景，导致去化出现困难，鼓励农民买房就是为了拉升需求并加速去化。

　　张波认为，农民进城买房的需求依然存在，内在推动因素包括对于优质教育资源的追求，对于更好公共配套资源的需求，但本轮政策对于农民推动的实际效果将低于市场预期。

　　陈霄也认为，从效果来看，可能实际作用有限，一方面农民进城多是短期务工，并且在农村也有固定居所，有限的优惠补贴力度尚难以触达农民，另一方面，三四线城市需求基本已经透支，通过这样的政策能增加的需求有限。

　　《经济日报》发布的评论文章指出，近期，我国部分城市推出鼓励农民进城买房的政策举措，多个市县提出给予购房农民税费减免、购房补贴等优惠。此类政策的初衷，一方面是为了应对房地产市场下行压力，挖掘农民购房潜力去库存，另一方面也意在试图“以房留人”，缓解当地城镇化率不足、人口流失的状况。

　　但是提升城市吸引力，是一项全方位系统工程，仅靠补贴等政策吸引农民进城购房，随着优惠效应的递减，恐怕难以真正留住农民，实现政策预期。农民愿不愿意进城，其“本”在于能不能获得就业机会，提高收入水平，在此基础上才愿意通过购房留下来。

　　此外，农民进城买房只是第一步，如果不下功夫完善产业配套设施和营商环境，健全就业、教育、医疗等保障，补齐环境基础设施、公共服务设施等方面短板，那么鼓励农民买房的效果就会大打折扣。

**Comment by ghost** [comment]
兄弟，你是贵州的吗，我看你上面写着中天吾乡

**Reply by ewkyrtas** [reply]
是的

**Reply by winnoryder** [reply]
共产党没有那么大的本事，所谓的喝茶大部分都是五毛虚张声势夸张出来的，当然也有部分使用大陆产的vpn导致的被定位，正常使用非大陆产vpn，共产党无法通过海外帐号就定位你的个人信息的

**Reply by naweiduxian** [reply]
一般来讲没人举报警察局根本不管的好吗，除非是VPN供应商被抓了把你供出来了
